<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTest</class>
 <widget class="QWidget" name="FormTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>325</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>10</number>
    </property>
    <widget class="FormTestProgram" name="tabProgram">
     <attribute name="title">
      <string>Program</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0"/>
    </widget>
    <widget class="FormTestVariable" name="tabVariable">
     <attribute name="title">
      <string>Variable</string>
     </attribute>
    </widget>
    <widget class="FormTestHome" name="tabHome">
     <attribute name="title">
      <string>Home</string>
     </attribute>
    </widget>
    <widget class="FormTestStatus" name="tabStatus">
     <attribute name="title">
      <string>Status</string>
     </attribute>
    </widget>
    <widget class="FormTestSDO" name="tabSDO">
     <attribute name="title">
      <string>SDO</string>
     </attribute>
    </widget>
    <widget class="FormUserParams" name="tabUserParams">
     <attribute name="title">
      <string>UserParams</string>
     </attribute>
    </widget>
    <widget class="FormRobotConf" name="tabRobotConf">
     <attribute name="title">
      <string>RobotConf</string>
     </attribute>
    </widget>
    <widget class="FormTestSave" name="tabSave">
     <attribute name="title">
      <string>Save</string>
     </attribute>
    </widget>
    <widget class="FormTestWorldZone" name="tabWorldZone">
     <attribute name="title">
      <string>WorldZone</string>
     </attribute>
    </widget>
    <widget class="FormTestInching" name="tabInching">
     <attribute name="title">
      <string>Inching</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabETC">
     <attribute name="title">
      <string>ETC</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnGetSlaveCount">
         <property name="text">
          <string>Get_SlaveCount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGetSlaveType">
         <property name="text">
          <string>Get_SlaveType</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGetSlaveModel">
         <property name="text">
          <string>Get_SlaveModel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="btnSetEMS">
         <property name="text">
          <string>Set_EMS_ON</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnRebootSystem">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>153</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Reboot_System</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_GetCurProgramStack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>169</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Get_CurProgramStack</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcInverse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>CalcInverse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEcatStats">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>EcatStats</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVersion">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>checkVersion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMultiData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>GetMultiData</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tapETC2">
     <attribute name="title">
      <string>ETC2</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>315</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="9" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="txtSetAutoRunValue"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="txtStableTime"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnGetDateTime">
         <property name="text">
          <string>Get_DateTime</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="txtGetAutoRunValue"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtLogCode"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtMoveToPos"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btnMoveToPos">
         <property name="text">
          <string>MoveToPosition</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txtDateTime_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnSetLog">
         <property name="text">
          <string>Set_Log</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="txtLogMsg"/>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="btnSetAutoRunSubPrg">
         <property name="text">
          <string>Set_AutoRunSubPrg</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="btnGetAutoRunSubPrg">
         <property name="text">
          <string>Get_AutoRunSubPrg</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btnSetDateTime">
         <property name="text">
          <string>Set_DateTime</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtDateTime"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Argment_1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Argment_2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnGetLog">
         <property name="text">
          <string>Get_Log</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Function_Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="btnSetAutoRunSubPrg_2">
         <property name="text">
          <string>get_crcErrorCount</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>200</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnClean">
          <property name="text">
           <string>Clean</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoNone</set>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="lineWrapColumnOrWidth">
        <number>0</number>
       </property>
       <property name="overwriteMode">
        <bool>false</bool>
       </property>
       <property name="tabStopWidth">
        <number>80</number>
       </property>
       <property name="acceptRichText">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormTestStatus</class>
   <extends>QWidget</extends>
   <header>formteststatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormUserParams</class>
   <extends>QWidget</extends>
   <header>formuserparams.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTestHome</class>
   <extends>QWidget</extends>
   <header>formtesthome.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormRobotConf</class>
   <extends>QWidget</extends>
   <header>formrobotconf.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTestWorldZone</class>
   <extends>QWidget</extends>
   <header location="global">formtestworldzone.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTestProgram</class>
   <extends>QWidget</extends>
   <header>formtestprogram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTestVariable</class>
   <extends>QWidget</extends>
   <header>formtestvariable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTestSDO</class>
   <extends>QWidget</extends>
   <header>formtestsdo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTestSave</class>
   <extends>QWidget</extends>
   <header>formtestsave.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTestInching</class>
   <extends>QWidget</extends>
   <header location="global">formtestinching.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
